### ISSUE TITLE
Master React, Quick Code tutorial. Part 1

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/react-only-13#2]
Link to the first part of tutorial 
https://medium.com/quick-code/lets-learn-react-chapter-1-setting-up-environment-e9505b6644

## Expected result

- [x] Read this tutorial chapter
- [x] follow code steps
- [x] move your code to this repository(at least one commit per day)
- [x] explain to me, what did you learn from it in a few sentences(not less than 5)
- [x] pass the test section
- [x] after reviewing your progress and closing this issue - you can move to the next chapter.

## Test
- [x] install raven package, install underscore package

#1

**Comment by khyati0212** [comment]
1. Installed node package create-react-app
2. Created a react application structure using create-react-app react-study
3. Removed the default CSS and test files from "src" folder which was displaying welcome page. 
4. Removed the import statements from the App.js and index.js related to the files we deleted in the above line.
5. Remove the default code from App.js and insert "Hello World" DIV. So here the react is rendering HTML code to JS and displaying on the local browser.
